src/app/GitUI/Translation/English.xlf

```diff
@@ -2166,6 +2166,10 @@ Suitable for some config files modified locally.</source>
   </file>
   <file datatype="plaintext" original="FolderBrowserButton" source-language="en">
     <body>
+      <trans-unit id="$this.Text">
+        <source>Browse...</source>
+        <target />
+      </trans-unit>
```

Comment:
> Regarding the “NO_TRANSLATE”: I understand what you mean by “noise in the translation file” but I’ve looked through them and I found that almost all of them are in dialogs that I’m still likely to fill with mnemonics in the future.

Such additional entries in the translation file are wrong because you have not added any new control.

Reply:
You’re absolutely right, I understand what you’re saying now. The changes have been made.